### PR TITLE
High alch value and profit to GE buy interface

### PR DESCRIPTION
**Pull request by bebeli555**
Adds high alch value and profit to the GE buy interface below the actively traded price on the grandexchange plugin. The profit thats in the brackets is calculated including the nature rune price so it shows how much you would make if you buy the item with the actively traded price and high alch it.
image

### CODE REVIEW

**Comment by austinmc32** [comment]
any updates on these? Id love to see this implemented

**Comment by Lord-Caim** [comment]
this would be great

**Comment by bebeli555** [comment]
I fixed and tested it now, its ready to merge, also set it's default option to be disabled
